resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/ResourceServicePersistenceProxy.java

```diff
@@ -113,10 +112,12 @@ public ResourceServicePersistenceProxy(final PersistenceLayer persistence, final
                                         1, persistNewResources(request)
                                 )))
                         )
-                )
-        ).mapMaterializedValue(ignored -> NotUsed.notUsed());
-
-        /**/
+                ))
+                .map((AuditableResourceResponse resource) -> {
+                    LOGGER.debug("Got resource {} for request {}", resource, request);
```

Comment:
Could a constant be done for this message as well so we pass in the relevant values?

Reply:
The others are  logging different stuff (id/type/format)
Similar to above, no static message, but put request at lower logging level and used static message for just that